Intermittent loop errors

I'm attempting to build something based on the octopi layout.

This build worked, but I wanted to remove the `if` on the workflow: https://github.com/myoung34/tilty-pi/actions/runs/187201484

which caused this failure despite no changes: https://github.com/myoung34/tilty-pi/runs/923707874

```
Resizing file system to 1620 MB...
++++ awk '{print $4-0}'
++++ sfdisk -d 2020-05-27-raspios-buster-lite-armhf.img
++++ grep 2020-05-27-raspios-buster-lite-armhf.img2
+++ start=532480
+++ offset=272629760
+++ detach_all_loopback 2020-05-27-raspios-buster-lite-armhf.img
++++ losetup
++++ awk '{ print $1 }'
++++ grep 2020-05-27-raspios-buster-lite-armhf.img
+++ for img in $(losetup  | grep $1 | awk '{ print $1 }' )
+++ losetup -d /dev/loop2
+ exit 1
losetup: /dev/loop2: detach failed: No such device or address
##[error]Process completed with exit code 1.
```

which seems to be from https://github.com/guysoft/CustomPiOS/blob/master/src/common.sh#L143

other intermittent failures:

 * https://github.com/myoung34/tilty-pi/runs/924344319
 * https://github.com/myoung34/tilty-pi/runs/924357321

Hit re-run enough times and finally there's a success with no changes: https://github.com/myoung34/tilty-pi/runs/924379003

--- a/src/common.sh
+++ b/src/common.sh
@@ -153,7 +153,9 @@ function detach_all_loopback(){
   # Cleans up mounted loopback devices from the image name
   # NOTE: it might need a better way to grep for the image name, its might clash with other builds
   for img in $(losetup  | grep $1 | awk '{ print $1 }' );  do
-    losetup -d $img
+    if [[ -f $img ]]; then
+    	losetup -d $img
+    fi
   done
 }
 

